molgenis-core-ui/src/main/resources/css/themes/bootstrap-molgenis.css

```diff
@@ -1,20 +0,0 @@
-.panel-title {
-    color: white;
-}
-
-.panel-heading {
-    background-color: #5B82A4;  
-}
-
-#plugin-container {
-	background: #F8FAFB;
-	padding: 5px;
-}
-
-label.horizontal-inline {
-	margin-right: 5px;
-}
-
-label.horizontal-inline.inbetween {
-	margin-left: 5px;
-}
```

Comment:
http://unminify.com/?
I was thinking the same thing but then i thought how little effort it is to unminify it.
Reason for removing is the ResourcePath object we use to generate a list of styles uses the molgenis.css as well as the generated molgenis.min.css

Creating two molgenis options in the user interface